tests/migrations/test_commands.py

```diff
@@ -898,6 +944,28 @@ def test_makemigrations_interactive_accept(self):
                 self.assertTrue(os.path.exists(merge_file))
             self.assertIn("Created new merge migration", out.getvalue())
 
+    def test_makemigrations_interactive_accept_when_rebasing(self):
+        """
+        makemigrations --rebase enters interactive mode and rebase properly.
+        """
+        # Monkeypatch interactive questioner to auto accept
+        with mock.patch('builtins.input', mock.Mock(return_value='y')):
+            out = io.StringIO()
+            with self.temporary_migration_module(module="migrations.test_migrations_conflict") as migration_dir:
+                call_command("makemigrations", "migrations", rebase=True, interactive=True, stdout=out)
+                # makemigrations --rebase does not guarantee any particular order of the rebased migrations.
```

Comment:
I wonder if I should provide something argument to decide which migration files are rebased. But I don't have nice idea.